src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -835,9 +835,19 @@ private function addValidationSection(ArrayNodeDefinition $rootNode)
                                 ->end()
                             ->end()
                         ->end()
-                        ->booleanNode('disable_not_compromised_password')
-                            ->defaultFalse()
-                            ->info('Disable NotCompromisedPassword Validator: the value will always be valid.')
+                        ->arrayNode('not_compromised_password')
+                            ->addDefaultsIfNotSet()
+                            ->info('Non-compromised password validator configuration')
```

Comment:
the validator is named `not compromised`, not `non-compromised`

Reply:
Done. I'd suggest do keep the variant with dash (`not-compromised`) in this human-readable description to make things clearer.